docker/sattle.cpp

```diff
@@ -0,0 +1,227 @@
+#include <pybind11/pybind11.h>
```

Comment:
Please ensure this file is documented with our C++ standards https://developer.lsst.io/cpp/api-docs.html